lighthouse/routes/eve_routes.py

```diff
@@ -0,0 +1,41 @@
+from http import HTTPStatus
+
+from flask import Blueprint, redirect, request
+from flask_cors import CORS
+
+bp = Blueprint("eve_routes", __name__)
+CORS(bp)
+
+
+def redirect_endpoint(base_url):
```

Comment:
Maybe add more comments here for someone new looking at this to figure out why/what we are doing.